src/NServiceBus.Newtonsoft.Json/NServiceBus.Newtonsoft.Json.csproj

```diff
@@ -2,23 +2,35 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>net452;netcoreapp2.0</TargetFrameworks>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>$(SolutionDir)NServiceBus.snk</AssemblyOriginatorKeyFile>
-    <DebugType>full</DebugType>
-    <DebugSymbols>True</DebugSymbols>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <OutputPath>..\..\binaries\</OutputPath>
-    <UpdateAssemblyInfo>true</UpdateAssemblyInfo>
     <NoWarn>0419</NoWarn>
-    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <Description>Newtonsoft.Json integration for NServiceBus.</Description>
```

Comment:
shouldn't this element be in the next property group? It seems like it's related to the nuget package?